src/CrudHandlingInterface.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+namespace Opstalent\CrudBundle;
+
+/**
+ * Class CrudHandlingInterface
+ * @author Szymon Kunowski <szymon.kunowski@gmail.com>
+ * @package Opstalent\CrudBundle
+ */
+abstract class CrudHandlingInterface
```

Comment:
I don't like this name. What do u think about `CrudRequestInterface` and moving it to `src/Request`?

Reply:
then `RequestListener` will be moved to `src/Request` too

Reply:
not sure if abstract class should be called `interface`

Reply:
I haven't noticed this :) but imo it should be interface, not a class at all

Reply:
i need this exact properties and functions (from this bundle perspective) so CrudRequestAbstract ?